Simplenote/src/main/res/values/dimens.xml

```diff
@@ -9,6 +9,7 @@
     <dimen name="padding_medium">12dp</dimen>
     <dimen name="padding_large">16dp</dimen>
     <dimen name="padding_extra_large">20dp</dimen>
+    <dimen name="padding_extra_extra_large">24dp</dimen>
```

Comment:
lol

Reply:
If you think that's funny, check out the [margins in WordPress-Android](https://github.com/wordpress-mobile/WordPress-Android/blob/d368609b3591e643c30e28ead8263970e1d2b777/WordPress/src/main/res/values/dimens.xml#L57-L72).

Reply:
`margin_extra_extra_medium_large` ftw